cmd/sonobuoy/app/delete.go

```diff
@@ -47,11 +49,19 @@ func NewCmdDelete() *cobra.Command {
 	AddRBACModeFlags(&deleteFlags.rbacMode, cmd.Flags(), DetectRBACMode)
 	AddDeleteAllFlag(&deleteopts.DeleteAll, cmd.Flags())
 	AddDeleteWaitFlag(&deleteFlags.wait, cmd.Flags())
+	AddWaitOutputFlag(&deleteFlags.waitOutput, cmd.Flags())
 
 	return cmd
 }
 
 func deleteSonobuoyRun(cmd *cobra.Command, args []string) {
+
+	var s *spinner.Spinner
```

Comment:
Dont need to define this here, just do `:=` below when you get the instance. Then we also wont need to add the import line at the top of this file.